### PR TITLE
fix: correct field name

### PR DESCRIPTION
**Pull request by ctron**
This will also be fixed on the backend side.

### CODE REVIEW

**Comment by ctron** [comment]
Ok, looks like it takes a bit more. @carlosthe19916 maybe you can finish this up, or start from scratch with this is a pointer.

File: client/src/app/api/models.ts [hunk 1]
```diff
@@ -185,14 +185,14 @@ export interface ImporterReport {
   report: {
```
Comment (gildub) [inline client/src/app/api/models.ts:185]: While at it, why not use interface definition for report instead ?
```ts
export interface ImporterReport {
  id: string;
  error?: string;
  report: ImporterReportDetails;
}
```

**Review by gildub — CHANGES_REQUESTED** [review]
The field name needs to be changed in `./src/app/pages/importer-list/importer-list.tsx`./src/app/pages/importer-list/importer-list.tsx too.

**Comment by carlosthe19916** [comment]
@gildub I pushed a commit to this branch to fix the missing change on ./src/app/pages/importer-list/importer-list.tsx .

**Review by carlosthe19916 — APPROVED** [review]

**Pull request merged**